interface/themes/colors/utilities/default_variables.scss

```diff
@@ -52,16 +52,7 @@ $theme-colors: (
   oe-palepale: $palepale,
   oe-palepurple: $palepurple,
   oe-paler: $paler,
-  oe-midnightblue: $midnightblue,
-  gray100: $gray-100,
-  gray200: $gray-200,
-  gray300: $gray-300,
-  gray400: $gray-400,
-  gray500: $gray-500,
-  gray600: $gray-600,
-  gray700: $gray-700,
-  gray800: $gray-800,
-  gray900: $gray-900
```

Comment:
Should be included and were there prior to new changes. These grays get used in some themes.

Reply:
Actually just below in theme color array:
```
$theme-colors: (
  primary: $blue,
  secondary: $dark,
  success: $green,
  info: $cyan,
  warning: $yellow,
  danger: $red,
  light: $gray-100,
  dark: $gray-800,
  oe-darkest: $darkest,
  oe-darker: $darker,
  oe-dark: $dark,
  oe-mid: $mid,
  oe-midpale: $midpale,
  oe-pale: $pale,
  oe-midpalepale: $midpalepale,
  oe-palepale: $palepale,
  oe-palepurple: $palepurple,
  oe-paler: $paler,
  oe-midnightblue: $midnightblue,
  gray100: $gray-100,
  gray200: $gray-200,
  gray300: $gray-300,
  gray400: $gray-400,
  gray500: $gray-500,
  gray600: $gray-600,
  gray700: $gray-700,
  gray800: $gray-800,
  gray900: $gray-900
);
```